### PR TITLE
Add an example of view

### PR DESCRIPTION
**Pull request by tony612**
Add an example of custom histogram buckets and renaming in prometheus. See https://github.com/open-telemetry/opentelemetry-go/issues/689

### CODE REVIEW

**Comment by dmathieu** [comment]
I wonder if we shouldn't keep the base example simple, to not confuse folks who just get started. How about adding a new example instead of extending the current one?

**Comment by tony612** [comment]
@dmathieu How about moving it to a new example `view` to demo how to use view? Cc @MrAlias

**Comment by XSAM** [comment]
> @dmathieu How about moving it to a new example `view` to demo how to use view? Cc @MrAlias

That would be good. 👍

File: example/view/main.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	"go.opentelemetry.io/otel/attribute"
+	otelprom "go.opentelemetry.io/otel/exporters/prometheus"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/sdk/instrumentation"
+	"go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/metric/aggregation"
+	"go.opentelemetry.io/otel/sdk/metric/view"
+)
+
+const meterName = "github.com/open-telemetry/opentelemetry-go/example/prometheus"
```
Comment (XSAM) [inline example/view/main.go:40]:
```suggestion
const meterName = "github.com/open-telemetry/opentelemetry-go/example/view"
```

**Review by XSAM — APPROVED** [review]

**Review by dmathieu — APPROVED** [review]

**Review by MrAlias — CHANGES_REQUESTED** [review]
Having this as a new module means it needs to be included in our dependabot config (should happen if you run `make`) and `versions.yaml`.

**Comment by tony612** [comment]
@MrAlias Please review again. Thanks.

File: example/view/main.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Default view to keep all instruments
+	defaultView, err := view.New(view.MatchInstrumentName("*"))
+	if err != nil {
+		log.Fatal(err)
+	}
```
Comment (MrAlias) [inline example/view/main.go:69]: Note, this shouldn't be needed. Tracking resolution with https://github.com/open-telemetry/opentelemetry-go/issues/3224

**Review by MrAlias — COMMENTED** [review]
Please add a changelog entry

**Review by MrAlias — APPROVED** [review]

**Pull request merged**